code/game/objects/structures/watercloset.dm

```diff
@@ -403,75 +403,7 @@
 		L.ExtinguishMob()
 		L.adjust_fire_stacks(-20) //Douse ourselves with water to avoid fire more easily
 		to_chat(L, "<span class='warning'>You've been drenched in water!</span>")
-		if(iscarbon(O))
-			var/mob/living/carbon/M = O
-			if(M.r_hand)
-				M.r_hand.clean_blood()
-			if(M.l_hand)
-				M.l_hand.clean_blood()
-			if(M.back)
-				if(M.back.clean_blood())
-					M.update_inv_back()
```

Comment:
You got rid of `back.clean_blood()` here but I don't see it added into `/human/clean_blood()`, which means backpacks won't get cleaned. Is that intended?

Reply:
Nope, not intended. Thanks nerd.